src/components/PageComponents/PageContainers.tsx

```diff
@@ -22,6 +22,8 @@ import styled from 'styled-components'
 import { fadeIn } from '@/animations'
 import { deviceBreakPoints } from '@/style/globalStyles'
 
+import Box from '../Box'
```

Comment:
It is actually VSCode that automatically imports it like this. It only happens when we are importing a component in a component within the component folder. It'd be quite annoying to then manually change automatic imports :/

Reply:
Yeah. Not a big deal.

Reply:
So, I figured it out, thanks for pushing me to research this in more detail!

There's a setting on VSCode that fixes that. I suggest we commit the `.vscode/settings.json` file so that we both share the same settings. So far, it only has 1 line, I suggest we add the second:

```json
{
  "typescript": {
    "tsdk": "node_modules/typescript/lib",
    "preferences.importModuleSpecifier": "non-relative"
  }
}
```


<img width="1124" alt="image" src="https://user-images.githubusercontent.com/1579899/214789682-0fd8e9e9-61e9-43f0-ae29-3afb93437957.png">

Reply:
I will update the import paths of every file in another PR so that this one doesn't grow and I don't get conflicts with the following ones if you don't mind!